build.sbt

```diff
@@ -226,11 +228,8 @@ lazy val storm = project.in(file("storm"))
 lazy val samza = project.in(file("samza"))
   .settings(commonSettings: _*)
   .settings(name := "tranquility-samza")
+  .settings(resolvers += "central-local" at "https://metamx.artifactoryonline.com/metamx/libs-releases-local")
```

Comment:
Is there a way to not have this in the build Def but still use it in the build process?